config/samples/sample-mpijob.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: pi
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  slotsPerWorker: 1
+  runPolicy:
+    cleanPodPolicy: Running
+    ttlSecondsAfterFinished: 60
```

Comment:
@tenzen-y is this implemented?

Reply:
Yes, the mpi-operator passes the parameter to batch/job for the launcher.

https://github.com/kubeflow/mpi-operator/blob/ccf2756f749336d652fa6b10a732e241a40c7aa6/pkg/controller/mpi_job_controller.go#L1366-L1389